File: CursoPOOUber/PHP/car.php
<?php
require_once('account.php');
class Car{
    public $id;
    public $license;
    public $drive;
    protected $passenger;

    public function __construct($license, $drive){
        $this->license = $license;
        $this->drive = $drive;
    }

    public function printDataCar() {
        echo "
        Licencia: $this->license 
        Drive: {$this->drive->name}
        Numero de pasajeros: $this->passenger
        "; 
    }

    public function getPassenger(){
        return $this->passenger;
    }

    public function setPassenger($passenger){
        if($passenger == 4){
            $this->passenger = $passenger;
        }else{
            echo "Necesitas asignar 4 pasajeros";
        }
    }
}

?>

File: CursoPOOUber/Java/Car.java
package Java;

public class Car {
    private Integer id;
    private String license;
    private Account drive;
    private Integer passenger;

    public Car(String license, Account drive){
        this.license = license;
        this.drive = drive;
    }

    void printDaTaCar(){
        if(passenger != null ){
            System.out.println("License: " + license + " Name Drive: " + drive.name + " Passengers: " + passenger);
        }
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDrive() {
        return drive;
    }

    public void setDrive(Account drive) {
        this.drive = drive;
    }

    public Integer getPassenger() {
        return passenger;
    }

    public void setPassenger(Integer passenger) {
        this.passenger = passenger;
    }
    
}
